administration/logging.md

```diff
@@ -22,6 +23,15 @@ Log files are written to either `userdata/log` (manual setup) or `/var/log/openh
 - `events.log`
 - `openhab.log`
 
+## openHABian Log Viewer (frontail)
+
+If you are running your openHAB instance with *openHABian* you are able to view your log entries through a nice and simple web interface.
+You can install it easily as an [optional component]({{base}}/installation/openhabian.html#optional-components) with the [openHABian configuration tool]({{base}}/installation/openhabian.html#openhabian-config).
+It can then be reached from [http://openhabianpi:9001/](http://openhabianpi:9001/) and gets new logs automatically.
```

Comment:
Consider breaking this into two sentences.  "It can then be reached from http://openhabianpi:9001/)."  "The tool gets new logs automatically."